src/config.ts

```diff
@@ -102,6 +112,13 @@ const config: Config = {
           address: "0x82af49447d8a07e3bd95bd0d56f35241523fbab1",
           decimals: 18
         }
+      },
+      rewardPoolIds: {
+        fxAUD: 4,
+        fxEUR: 7,
+        fxPHP: 5,
+        fxUSD: 8,
```

Comment:
```suggestion
        fxAUD: 4,
        fxEUR: 7,
        fxPHP: 5,
        fxUSD: 8,
```
These should be more explicit that it is for keeper pools